### PR TITLE
Cleans up imports across all modules

### LINKED ISSUE #895: Clean up project imports
Discovered while troubleshooting VSTS-CI builds: there are many unused and unsorted imports in the codebase. Use Android Studio's import optimizer to tidy these up.

### PR DESCRIPTION
**Pull request by iambmelt**
This change cleans and sorts imports across all ADAL project modules. It closes #895

### CODE REVIEW

File: adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java [hunk 1]
```diff
@@ -23,13 +23,20 @@
...
+import android.content.Context;
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.test.ActivityUnitTestCase;
+import android.test.RenamingDelegatingContext;
+import android.test.UiThreadTest;
+import android.test.suitebuilder.annotation.SmallTest;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
```
Comment (heidijinxujia) [inline adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java]: Are they the same? It looks like they just swapped.

**Comment by heidijinxujia** [comment]
I think I am kind of confused here, I saw a lot of imports are just re-ordered. Is it necessary?

File: adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java (hunk 1 above)
Comment (iambmelt) [inline adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java]: Should be the same, but alphabetized -- the output of running Android Studio's import optimizer

**Comment by heidijinxujia** [comment]
OK. Got it.

**Comment by heidijinxujia** [comment]
As to Android Studio's import optimizer, is there a way we could run it automatically per build in Android Studio? Or this is an one-time task,

**Comment by iambmelt** [comment]
@heidijinxujia After a little sleuthing, it turns out the answer is yes! And it doesn't even mean setting up a `git` pre-commit hook 😎 

Check out `File | Settings | Editor | General | Auto Import | Optimize imports on the fly`
imports

**Pull request merged**